### PR TITLE
Update creators.yml

### PR DESCRIPTION
**Pull request by appdesigndev**
New creator on creators.yml

### CODE REVIEW

**Comment by laurieontech** [comment]
It doesn't look as though this is a Gatsby site so I'm going to close this PR.

File: docs/creators/creators.yml [hunk 1]
```diff
@@ -478,3 +478,14 @@
...
   hiring: false
   portfolio: true
+- name: App Design
+  type: agency
+  description: >-
+    My name is Oliver. I am a web developer and app designer.
+  location: Canada
+  website: "https://appdesign.dev/"
+  github: "https://github.com/appdesigndev"
+  image: images/app-design-dev.jpeg
```
Comment (pieh) [inline docs/creators/creators.yml]: This image wasn't added, so I do need to revert this PR for now ( https://github.com/gatsbyjs/gatsby/pull/21223 ). Please re-open and the image ;)

Comment (sidharthachatterjee) [reply docs/creators/creators.yml]: Oops

Comment (appdesigndev) [reply docs/creators/creators.yml]: it say: "You’re not authorized to merge this pull request." I need Code owner review required, thanks :)

**Pull request merged**